woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -55,6 +55,40 @@ def test_ordinal_init_with_order():
     assert ordinal_from_tuple.order == order
 
 
+def test_ordinal_transform_validates(ordinal_transform_series_pandas) -> None:
+    typ = Ordinal(order=None)
+    with pytest.raises(TypeError):
```

Comment:
Should we also check that the error message matches what we expect, so we know the right `TypeError` is getting raised?

Reply:
Yeah, probably a good idea.  I'll change it.